assets/airports/kdca.json

```diff
@@ -6,673 +6,1856 @@
     },
     "icao": "KDCA",
     "iata": "DCA",
-    "magnetic_north": -10.8,
-    "ctr_radius": 50,
-    "ctr_ceiling": 18000,
-    "initial_alt": 5000,
-    "position": ["N38.852222", "W77.037778"],
+    "magnetic_north": 9,
+    "ctr_radius": 80,
+    "ctr_ceiling": 7000,
+    "initial_alt": 3000,
+    "position": ["N38d51m05.20", "W077d02m15.80", "14.1ft"],
     "rr_radius_nm": 5.0,
-    "rr_center": ["N38.852222", "W77.037778"],
-    "has_terrain": false,
+    "rr_center": ["N38d51m05.20", "W077d02m15.80"],
+    "has_terrain": true,
     "wind": {
-        "angle": 350,
-        "speed": 7
+        "angle": 217,
```

Comment:
Since we don't have RNAV procedures yet, approaches to the 19 are basically impossible. It'd be better for now if we used a north configuration.

Reply:
Fixed 👍

Reply:
this is _okay_ if you want to continue anyway.